xbmc/guilib/GUIFontManager.cpp

```diff
@@ -120,8 +125,18 @@ CGUIFont* GUIFontManager::LoadTTF(const std::string& strFontName, const std::str
 
   // Check if the file exists, otherwise try loading it from the global media dir
   std::string file = URIUtils::GetFileName(strFilename);
-  if (!CheckFont(strPath,"special://home/media/Fonts",file))
-    CheckFont(strPath,"special://xbmc/media/Fonts",file);
+  if (!CheckFont(strPath, "special://home/media/Fonts", file))
+    if (!CheckFont(strPath, "special://xbmc/media/Fonts", file))
```

Comment:
Missing bracket?

Reply:
Btw combine both of them into one if statement

Reply:
if you say so. it's hard to remember which project has which brace rules, at work i am not allowed to use those braces in this case...

Reply:
braced, combined.

Reply:
non-single lines of code should be surrounded by braces if you asked me, it's better for readability.